auth/src/main/java/io/camunda/zeebe/auth/impl/JwtAuthorizationEncoder.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.auth.impl;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTCreationException;
+import io.camunda.zeebe.auth.api.AuthorizationEncoder;
+import io.camunda.zeebe.auth.api.JwtAuthorizationBuilder;
+import io.camunda.zeebe.util.exception.UnrecoverableException;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JwtAuthorizationEncoder
+    implements JwtAuthorizationBuilder<JwtAuthorizationEncoder, Algorithm, String>,
+        AuthorizationEncoder {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(JwtAuthorizationEncoder.class);
+  private String issuer = "zeebe-gateway";
+  private String audience = "zeebe-broker";
+  private String subject = "Authorization";
```

Comment:
🔧 Perhaps we could define these defaults in JwtAuthorizationBuilder instead, that way we only need to define them ones instead of both in the encoder and decoder. It's also clear to users of the interface what the defaults are.

Reply:
:+1: That's a good point. I'll apply it in the next PR that makes related changes.